### ISSUE TITLE
GAPIS Panic index out of range in ResourceDataResolvable

### ISSUE THREAD

**Issue by pmuetschard** [google/gapid#1477]
GAPID Version: 0.9.4:549:e4ca278a22b8bdfab6503b7126055e8984115cb7
OS: mac os x x86_64

Tracing minecraft on S8.

```
panic: runtime error: index out of range [recovered]
	panic: 
--- *resolve.AllResourceDataResolvable ---
 after:<capture:<id:<data:"\273\346\252\037\337\005\324Ys\253\251\034CE\2050\360\353\265^" > > indices:5353 > 
 Store():
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:50  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1

 Build() #0:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #1:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #2:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #3:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #4:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #5:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #6:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #7:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #8:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #9:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 ... 124 more Build() calls (truncated)

--- *resolve.ResourceDataResolvable ---
 path:<id:<data:"\256\177\224\2542\332\253\362\243\326\220\277\304\3443\004\000\343\212=" > after:<capture:<id:<data:"\273\346\252\037\337\005\324Ys\253\251\034CE\2050\360\353\265^" > > indices:5353 > > 
 Store():
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:50  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:119  github.com/google/gapid/gapis/resolve.ResourceData
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:308  github.com/google/gapid/gapis/resolve.ResolveInternal
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:249  github.com/google/gapid/gapis/resolve.ResolveService
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:35  github.com/google/gapid/gapis/resolve.(*GetResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1

 Build() #0:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:119  github.com/google/gapid/gapis/resolve.ResourceData
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:308  github.com/google/gapid/gapis/resolve.ResolveInternal
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:249  github.com/google/gapid/gapis/resolve.ResolveService
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:35  github.com/google/gapid/gapis/resolve.(*GetResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #1:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:119  github.com/google/gapid/gapis/resolve.ResourceData
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:308  github.com/google/gapid/gapis/resolve.ResolveInternal
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/thumbnail.go:106  github.com/google/gapid/gapis/resolve.ResourceDataThumbnail
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/thumbnail.go:37  github.com/google/gapid/gapis/resolve.Thumbnail
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:324  github.com/google/gapid/gapis/resolve.ResolveInternal
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:249  github.com/google/gapid/gapis/resolve.ResolveService
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:35  github.com/google/gapid/gapis/resolve.(*GetResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve

--- *resolve.GetResolvable ---
 path:<resource_data:<id:<data:"\256\177\224\2542\332\253\362\243\326\220\277\304\3443\004\000\343\212=" > after:<capture:<id:<data:"\273\346\252\037\337\005\324Ys\253\251\034CE\2050\360\353\265^" > > indices:5353 > > > 
 Store():
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:50  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:27  github.com/google/gapid/gapis/resolve.Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/server/server.go:259  github.com/google/gapid/gapis/server.(*server).Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/server/grpc.go:167  github.com/google/gapid/gapis/server.(*grpcServer).Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/service/service.pb.go:5902  github.com/google/gapid/gapis/service._Gapid_Get_Handler.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/auth/auth.go:110  github.com/google/gapid/core/app/auth.ServerInterceptor.func1
  /tmpfs/src/github/src/github.com/google/gapid/gapis/service/service.pb.go:5904  github.com/google/gapid/gapis/service._Gapid_Get_Handler
  /tmpfs/src/github/src/github.com/google/gapid/third_party/src/google.golang.org/grpc/server.go:719  google.golang.org/grpc.(*Server).processUnaryRPC
  /tmpfs/src/github/src/github.com/google/gapid/third_party/src/google.golang.org/grpc/server.go:905  google.golang.org/grpc.(*Server).handleStream

 Build() #0:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:27  github.com/google/gapid/gapis/resolve.Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/server/server.go:259  github.com/google/gapid/gapis/server.(*server).Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/server/grpc.go:167  github.com/google/gapid/gapis/server.(*grpcServer).Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/service/service.pb.go:5902  github.com/google/gapid/gapis/service._Gapid_Get_Handler.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/auth/auth.go:110  github.com/google/gapid/core/app/auth.ServerInterceptor.func1
  /tmpfs/src/github/src/github.com/google/gapid/gapis/service/service.pb.go:5904  github.com/google/gapid/gapis/service._Gapid_Get_Handler
  /tmpfs/src/github/src/github.com/google/gapid/third_party/src/google.golang.org/grpc/server.go:719  google.golang.org/grpc.(*Server).processUnaryRPC
  /tmpfs/src/github/src/github.com/google/gapid/third_party/src/google.golang.org/grpc/server.go:905  google.golang.org/grpc.(*Server).handleStream
 [recovered]
	panic: 
--- *resolve.AllResourceDataResolvable ---
 after:<capture:<id:<data:"\273\346\252\037\337\005\324Ys\253\251\034CE\2050\360\353\265^" > > indices:5353 > 
 Store():
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:50  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1

 Build() #0:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #1:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #2:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #3:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #4:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #5:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #6:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #7:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #8:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #9:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:128  github.com/google/gapid/gapis/resolve.(*ResourceDataResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 ... 124 more Build() calls (truncated)

--- *resolve.ResourceDataResolvable ---
 path:<id:<data:"\256\177\224\2542\332\253\362\243\326\220\277\304\3443\004\000\343\212=" > after:<capture:<id:<data:"\273\346\252\037\337\005\324Ys\253\251\034CE\2050\360\353\265^" > > indices:5353 > > 
 Store():
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:50  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:119  github.com/google/gapid/gapis/resolve.ResourceData
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:308  github.com/google/gapid/gapis/resolve.ResolveInternal
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:249  github.com/google/gapid/gapis/resolve.ResolveService
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:35  github.com/google/gapid/gapis/resolve.(*GetResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1

 Build() #0:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:119  github.com/google/gapid/gapis/resolve.ResourceData
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:308  github.com/google/gapid/gapis/resolve.ResolveInternal
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:249  github.com/google/gapid/gapis/resolve.ResolveService
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:35  github.com/google/gapid/gapis/resolve.(*GetResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:214  github.com/google/gapid/gapis/database.(*memory).resolveLocked.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/crash/crash.go:65  github.com/google/gapid/core/app/crash.Go.func1
 Build() #1:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resource_data.go:119  github.com/google/gapid/gapis/resolve.ResourceData
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:308  github.com/google/gapid/gapis/resolve.ResolveInternal
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/thumbnail.go:106  github.com/google/gapid/gapis/resolve.ResourceDataThumbnail
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/thumbnail.go:37  github.com/google/gapid/gapis/resolve.Thumbnail
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:324  github.com/google/gapid/gapis/resolve.ResolveInternal
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/resolve.go:249  github.com/google/gapid/gapis/resolve.ResolveService
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:35  github.com/google/gapid/gapis/resolve.(*GetResolvable).Resolve
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/memory.go:127  github.com/google/gapid/gapis/database.(*record).resolve

--- *resolve.GetResolvable ---
 path:<resource_data:<id:<data:"\256\177\224\2542\332\253\362\243\326\220\277\304\3443\004\000\343\212=" > after:<capture:<id:<data:"\273\346\252\037\337\005\324Ys\253\251\034CE\2050\360\353\265^" > > indices:5353 > > > 
 Store():
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:50  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:27  github.com/google/gapid/gapis/resolve.Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/server/server.go:259  github.com/google/gapid/gapis/server.(*server).Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/server/grpc.go:167  github.com/google/gapid/gapis/server.(*grpcServer).Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/service/service.pb.go:5902  github.com/google/gapid/gapis/service._Gapid_Get_Handler.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/auth/auth.go:110  github.com/google/gapid/core/app/auth.ServerInterceptor.func1
  /tmpfs/src/github/src/github.com/google/gapid/gapis/service/service.pb.go:5904  github.com/google/gapid/gapis/service._Gapid_Get_Handler
  /tmpfs/src/github/src/github.com/google/gapid/third_party/src/google.golang.org/grpc/server.go:719  google.golang.org/grpc.(*Server).processUnaryRPC
  /tmpfs/src/github/src/github.com/google/gapid/third_party/src/google.golang.org/grpc/server.go:905  google.golang.org/grpc.(*Server).handleStream

 Build() #0:
  /tmpfs/src/github/src/github.com/google/gapid/gapis/database/database.go:54  github.com/google/gapid/gapis/database.Build
  /tmpfs/src/github/src/github.com/google/gapid/gapis/resolve/get.go:27  github.com/google/gapid/gapis/resolve.Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/server/server.go:259  github.com/google/gapid/gapis/server.(*server).Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/server/grpc.go:167  github.com/google/gapid/gapis/server.(*grpcServer).Get
  /tmpfs/src/github/src/github.com/google/gapid/gapis/service/service.pb.go:5902  github.com/google/gapid/gapis/service._Gapid_Get_Handler.func1
  /tmpfs/src/github/src/github.com/google/gapid/core/app/auth/auth.go:110  github.com/google/gapid/core/app/auth.ServerInterceptor.func1
  /tmpfs/src/github/src/github.com/google/gapid/gapis/service/service.pb.go:5904  github.com/google/gapid/gapis/service._Gapid_Get_Handler
  /tmpfs/src/github/src/github.com/google/gapid/third_party/src/google.golang.org/grpc/server.go:719  google.golang.org/grpc.(*Server).processUnaryRPC
  /tmpfs/src/github/src/github.com/google/gapid/third_party/src/google.golang.org/grpc/server.go:905  google.golang.org/grpc.(*Server).handleStream
```

**Comment by ben-clayton** [comment]
I think this crash is missing information from the end of the log. Was there any more printed?

**Comment by ben-clayton** [comment]
Fixed by #1480
